atat/src/ingress_manager.rs

```diff
@@ -84,6 +84,8 @@ impl IngressManager {
     /// raw bytes being the core type allows the ingress manager to
     /// be abstracted over the communication medium.
     pub fn write(&mut self, data: &[u8]) {
+        #[cfg(feature = "logging")]
+        log::trace!("Ingress: Receiving {} bytes", data.len());
```

Comment:
I am not sure this is a good place to have this, as write is usually called from interrupts, meaning it is a bit timing sensitive.

Reply:
True, although you will most probably only activate logging on platforms like Linux, where it doesn't run in an ISR.

Also, the logging isn't that useful anymore if you don't see processed bytes. Do you have a better idea where to place it?

Reply:
Hmm. I guess you make a good point. 
I would have liked something that can also be used to debug directly on micro controllers, as not everyone has the option of doing it in linux.. But i'd be happy to settle with this for now, and if we come up with a better way down the road, it can be changed.